ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -32,6 +32,7 @@ class FaultInjector
   // Output /faults/joint_states, a modified version of /joint_states, injecting
   // simple message faults that don't need to be simulated at their source.
   void jointStateCb(const sensor_msgs::JointStateConstPtr& msg);
+  void testFxn(const sensor_msgs::JointStateConstPtr& msg);
```

Comment:
I understand that's this is a test but I think you can come up with a more descriptive name.
What is Fxn anyway!

Reply:
fxn is shorthand for function. And yea I can remove this, I think we should delete

Reply:
Never seen that one, and it confused me too.   "Fn" has been a standard abbreviation for ages.  I would naturally read "testFn" as "test function".